web-client/public/js/setup-load-and-import-handlers.js

```diff
@@ -167,9 +168,11 @@ export const setupLoadAndImportHandlers = grnState => {
         $(demoClass).on("click", () => {
             loadDemo(demoPath, demoClass, demoName);
         });
-
         $("#demoSourceDropdown").on("change", () => {
-            loadDemo(demoPath, demoClass, demoName);
+            let selected = `.${$("#demoSourceDropdown").val()}`;
```

Comment:
When you know for sure that you won’t need to reassign a variable, default to `const` instead of `let`
```suggestion
            const selected = `.${$("#demoSourceDropdown").val()}`;
```